classes/Product.php

```diff
@@ -5194,10 +5188,10 @@ public static function addCustomizationPrice(&$products, &$customized_datas)
                 $product_update['customizationQuantityReturned'] = $customization_quantity_returned;
 
                 if ($customization_quantity) {
-                    $product_update['total_wt'] = $price_wt * ($product_quantity - $customization_quantity);
-                    $product_update['total_customization_wt'] = $price_wt * $customization_quantity;
-                    $product_update['total'] = $price * ($product_quantity - $customization_quantity);
-                    $product_update['total_customization'] = $price * $customization_quantity;
+                    $product_update['total_wt'] = $product_update['unit_price_tax_incl'] * $product_quantity ;
+                    $product_update['total_customization_wt'] = $product_update['unit_price_tax_incl'] * $customization_quantity;
```

Comment:
$product_update['total_customization_wt'] is defined only in one template
./admindev/themes/default/template/controllers/carts/helpers/view/view.tpl:109
$product_update['total_customization_wt'] = $product_update['total_wt']
So I suggest to to replace in view.tpl by total_wt and remove $product_update['total_customization_wt']. But since 1.7 this part of template is not used because the cart BO page use PrestaShopBundle...